web/src/app/monitors/monitors-list/monitors-list.component.html

```diff
@@ -16,13 +16,23 @@
     <mat-card-content>
       <mat-list>
         <mat-list-item fxLayout="column" *ngFor="let monitor of monitors">
-          <h4 mat-line><a [routerLink]="[monitor.uid,'pings']">{{monitor.name}}</a></h4>
-          <p mat-line>{{monitor.method}} {{monitor.path}} {{monitor.expectedCode}} {{monitor.expectedText}}</p>
-          <p class="button-right">
-            <button mat-raised-button color="accent" [disabled]= "manualPing" (click)="pingMonitor(monitor.uid)">Ping</button>
-            <button mat-raised-button color="accent" [routerLink]="monitor.uid">Edit</button>
-            <button mat-raised-button color="warn" (click)="deleteMonitor(monitor.uid)">Delete</button>
-          </p>
+          <div class="monitors">
+            <h4 mat-line><a [routerLink]="[monitor.uid,'pings']">{{monitor.name}}</a></h4>
+            <p mat-line>{{monitor.method}} {{monitor.path}} {{monitor.expectedCode}} {{monitor.expectedText}}</p>
+            <p mat-line>Successful pings: {{monitor.successPingCounts ? monitor.successPingCounts : 0}} | Unsuccessful pings: {{monitor.unsuccessPingCounts ? monitor.unsuccessPingCounts : 0}}  </p>
+            <p mat-line></p>
+          </div>
+          <div class="pings" fxLayout="column">
```

Comment:
Latest ping is not always available. Suggest...


```suggestion
          <div class="pings" fxLayout="column" *ngIf="monitor.latestPing">
```

Reply:
When there is no latest ping, the action buttons move - suggest using flexbox here

<img width="1393" alt="Screenshot 2019-07-21 18 15 00" src="https://user-images.githubusercontent.com/624760/61594418-7e83bf00-abe3-11e9-9b7d-a48e6fa33395.png">